src/app/_components/NewsletterForm.tsx

```diff
@@ -49,12 +76,19 @@ export function NewsletterForm() {
             error={getError(methods.formState.errors, 'email')}
           />
         </div>
-        <div className="flex min-w-44 [&>*:first-child]:flex-1">
+        <div className="min-w-44 flex-shrink-0 flex-grow">
```

Comment:
@barbaraperic - I changed `[&>*:first-child]:flex-1` because this, to me, is still "reaching into the component."

Reply:
Oh 😞 I though this worked. Sorry. I can have a second look..